### PR TITLE
Revise rules for the use of enable and disable

### PR DESCRIPTION
**Pull request by cwillum**
Signed-off-by: cwillum <cwmmoore@amazon.com>

### Description
This change to the style guide and terms glossary makes room for the use of the words *enable* and *disable* to describe making a feature or command either available or unavailable. 

### Issues Resolved
This changes guidance so that writers can use the terms enable and disable to talk about features as available or unavailable.
Guidance for avoiding the use of these terms to describe a person's ability to do something (or lack of ability) in TERMS.md and STYLE_GUIDE.md still exists and remains valid.

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Review by hdhalter — COMMENTED** [review]
Out of curiosity, why are you promoting this change? Is not what we have sufficient? The alternatives provided are bias-free terms.

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -427,8 +427,8 @@ The following terms may be problematic *in some contexts*. This doesn’t mean t
...
-| enable                   | Use *turn on*, *activate*, or *start* instead of *enable* to support bias-free documentation, when possible. Otherwise, use *enable* to describe making a feature or command available. If the UI uses *active* and *inactive* to describe these states, use *activate* and *deactivate* in the documentation. <br><br> Avoid using *enable* to refer to making something possible for the user. Instead, rewrite to focus on what's important from the user's point of view. For example, “With ABC, you can do XYZ” is a stronger statement than “ABC enables you to XYZ.” Additionally, using a task-based statement is usually more clear than the vague “…enables you to….” |
+| disable                  | Don't use *disable* to refer to users. You can use *disable* to describe making a feature or command unavailable.                                    |
```
Comment (natebower) [inline STYLE_GUIDE.md:430]:
```suggestion
| disable                  | You can use *disable* to describe making a feature or command unavailable. Don't use *disable* to refer to users. |
```

File: STYLE_GUIDE.md [hunk 2]
```diff
@@ -427,8 +427,8 @@ The following terms may be problematic *in some contexts*. This doesn’t mean t
...
+| enable                   | Avoid using *enable* to refer to making something possible for the user. Instead, rewrite to focus on what's important from the user's point of view. For example, “With ABC, you can do XYZ” is a stronger statement than “ABC enables you to XYZ.” Additionally, using a task-based statement is usually more clear than the vague “…enables you to….” <br><br> You can use *enable* to describe making a feature or command available. |
```
Comment (natebower) [inline STYLE_GUIDE.md:431]:
```suggestion
| enable                   | You can use *enable* to describe making a feature or command available. Avoid using *enable* to refer to making something possible for the user. Instead, rewrite to focus on what's important from the user's point of view. For example, “With ABC, you can do XYZ” is a stronger statement than “ABC enables you to XYZ.” Additionally, using a task-based statement is usually more clear than the vague “…enables you to….” |
```

File: TERMS.md [hunk 3]
```diff
@@ -203,7 +203,14 @@ Use to describe a list of items that aren’t allowed (blocked). Do not use _bla
 
 **disable**
 
-Use *turn off*, *deactivate*, or *stop* instead of *disable* to support bias-free documentation, when possible. Otherwise, use *disable* to describe making a feature or command unavailable. If the UI uses *active* and *inactive* to describe these states, use *activate* and *deactivate* in the documentation. Don't use *disable* to refer to users.
+Use *disable* to describe making a feature or command unavailable. Examples:
```
Comment (natebower) [inline TERMS.md:206]:
```suggestion
Use *disable* to describe making a feature or command unavailable. For example:
```

File: TERMS.md [hunk 4]
```diff
@@ -203,7 +203,14 @@ Use to describe a list of items that aren’t allowed (blocked). Do not use _bla
...
+
+- Clear the checkbox to disable automatic monitoring.
+- The feature is disabled by default.
+
+Note that other alternative terms to *disable*—such as *deactivate*, *turn off*, or *stop*—are acceptable usage where appropriate and may be found in existing documentation. One important consideration when choosing terminology for these cases is that you make sure the language matches corresponding language in the user interface, if applicable.
```
Comment (natebower) [inline TERMS.md:211]:
```suggestion
Note that alternatives to *disable*—such as *deactivate*, *turn off*, or *stop*—are acceptable usage where appropriate and may be found in existing documentation. In all cases, use language that corresponds to the language used in the UI, if applicable.
```

File: STYLE_GUIDE.md (hunk 2 above)
Comment (natebower) [reply STYLE_GUIDE.md:431]: For these two, I think it makes sense for the "can" statements to come first so as to emphasize them.

File: TERMS.md [hunk 5]
```diff
@@ -233,7 +240,12 @@ Use as a singular noun or adjective to refer to the collective concept, and use
 
 **enable**
 
-Use _turn on_ or _activate_ instead of *enable* to support bias-free documentation, when possible. Otherwise, use *enable* to describe making a feature or command available. If the UI uses *active* and *inactive* to describe these states, use *activate* and *deactivate* in the documentation.
+Use *enable* to describe making a feature or command available. Examples:
```
Comment (natebower) [inline TERMS.md:243]:
```suggestion
Use *enable* to describe making a feature or command available. For example:
```

File: TERMS.md [hunk 6]
```diff
@@ -233,7 +240,12 @@ Use as a singular noun or adjective to refer to the collective concept, and use
...
+
+- Select the checkbox to enable automatic monitoring.
+- The feature is enabled by default.
+
+Note that other alternative terms to *enable*—such as *activate*, *turn on*, *start*—are acceptable usage where appropriate and may be found in existing documentation. One important consideration when choosing terminology for these cases is that you make sure the language matches corresponding language in the user interface, if applicable.
```
Comment (natebower) [inline TERMS.md:248]:
```suggestion
Note that alternatives to *enable*—such as *activate*, *turn on*, or *start*—are acceptable usage where appropriate and may be found in existing documentation. In all cases, use language that corresponds to the language used in the UI, if applicable.
```

**Review by natebower — APPROVED** [review]
@cwillum Approved with these changes. Thanks for raising this and suggesting the improvements!

File: TERMS.md (hunk 6 above)
Comment (cwillum) [reply TERMS.md]: Much better. Thanks for the review.

File: TERMS.md (hunk 4 above)
Comment (cwillum) [reply TERMS.md]: Much better.

File: STYLE_GUIDE.md (hunk 2 above)
Comment (cwillum) [reply STYLE_GUIDE.md]: Agreed.

File: STYLE_GUIDE.md (hunk 1 above)
Comment (cwillum) [reply STYLE_GUIDE.md]: Agreed.

**Review by cwillum — COMMENTED** [review]
Thanks for the review. Incorporated all suggested changes.

**Comment by cwillum** [comment]
@hdhalter The idea behind this change was to preserve guidance that discourages writers from using *enable* or *disable* to describe users themselves (thereby honoring bias-free content) while removing any constraints on using the terminology to describe the state of a feature—either available or unavailable. In my own opinion, and from what I've gathered from others, these words are the perfect tools for a specific job. In many cases, the alternatives are awkward and at best adequate at explaining the condition of a feature. Still, there might be other opinions and arguments against this. We can discuss further and make additional changes if need be.

**Pull request merged**